packages/vscode-extension/src/webview/components/UrlSelect.tsx

```diff
@@ -15,14 +15,15 @@ const SelectItem = React.forwardRef<HTMLDivElement, PropsWithChildren<Select.Sel
 );
 
 interface UrlSelectProps {
+  key: number;
   value: string;
   onValueChange: (newValue: string) => void;
   recentItems: UrlItem[];
   items: UrlItem[];
   disabled?: boolean;
 }
 
-function UrlSelect({ onValueChange, recentItems, items, value, disabled }: UrlSelectProps) {
+function UrlSelect({ onValueChange, recentItems, items, key, value, disabled }: UrlSelectProps) {
```

Comment:
I believe that it only has been added in React 19 that you can receive key prop like this. I don't think in the current version in propagates down to the component